main.cpp

```diff
@@ -135,25 +190,25 @@ int main()
     //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
     auto carRented = rentACar(6, 2); 
     
-    //1)
-    
-    //2)
+    auto Car = carCost(5000.f, 20000.f, 1);
     
-    //3)
+    char Pass = password('j', 'm');
+    
```

Comment:
variable names should begin with a lowercase first letter.
Type names should begin with an uppercase first letter
review the coding standard found in the Readme.md file
- [ ] revise